modules/ROOT/pages/configure-ingress-http-resource.adoc

```diff
@@ -0,0 +1,228 @@
+= Configuring Ingress with HTTP Route-Template Custom Resource in Runtime Fabric
+
+Configure ingress support for Mule apps on the Anypoint Runtime Fabric platform using the HTTP route-template custom resource.
+
+== Benefits of Ingress Configuration with the HTTP Route-Template Custom Resource
+
+Current ingress support for Mule apps on the Runtime Fabric platform is confined to https://kubernetes.io/docs/concepts/services-networking/ingress/[Kubernetes ingress] resources, which restrict you from using cloud vendor-native HTTP routing solutions, such as OpenShift Routes on OpenShift clusters, Istio, or Service Mesh. To address this limitation, Runtime Fabric supports a custom resource `httproutetemplates.rtf.mulesoft.com` template.
+
+Use the HTTP route-template custom resource in Runtime Fabric to define and implement custom routing configurations. With this approach, you can take advantage of compatibility with a range of ingress controllers and cloud-native routing mechanisms, including support for:
+
+* Wildcard for host-based routing.
+* Creation of `httproutetemplates.rtf.mulesoft.com` templates to map a set of application endpoints to arbitrary HTTP routing resources for path-based routing..
+* Synchronization of domains or hosts from the HTTP route templates with the Anypoint control plane.
+* Using parameter placeholders in `httproutetemplates.rtf.mulesoft.com` templates for additional extensibility.
+
+== HTTP Route-Template Custom Resource Placeholders
+
+Runtime Fabric reveals placeholder properties that cluster administrators can use in their HTTP route-template custom resource. These placeholders are evaluated and replaced with actual data by Runtime Fabric when rendering the deployment charts.
```

Comment:
"reveals" is what writer suggested to me, but I wonder if that's correct. Maybe "provides" is a better word?